app/templates/custom-elements/remote-screen.html

```diff
@@ -199,6 +190,107 @@
           screenElement.addEventListener("mousedown", (evt) => {
             this.rateLimitedMouse.onMouseDown(evt);
           });
+          //-------------------------------------------------------------------------------
+          var isPointerMoving = false;
+          var touchStartTimestamp = null;
+          var longTouchDuration = 600; // Time in milliseconds for a touch to be considered a long touch
```

Comment:
Is this an arbitrary decision we're making? Or does this match a convention elsewhere? We should just document whether it's us making the decision or if this is defined somewhere.

Reply:
That is an arbitrary decision.